google/cloud/bigtable/tools/run_emulator_utils.sh

```diff
@@ -34,22 +41,22 @@ function start_emulators {
 
   # The tests typically run in a Docker container, where the ports are largely
   # free; when using in manual tests, you can set EMULATOR_PORT.
-  readonly PORT=${EMULATOR_PORT:-9000}
-  "${CBT_EMULATOR_CMD}" -port "${PORT}" >emulator.log 2>&1 </dev/null &
+  local emulator_port=${EMULATOR_PORT:-9000}
```

Comment:
Was this supposed to remain readonly?  If so, I think you missed the `-r` after `local`.